scanpopup.cc

```diff
@@ -860,9 +860,10 @@ void ScanPopup::typingEvent( QString const & t )
   }
   else
   {
-    ui.translateBox->translateLine()->setFocus();
-    ui.translateBox->setText( t, true );
-    ui.translateBox->translateLine()->setCursorPosition( t.size() );
+      ui.translateBox->translateLine()->clear();
+      ui.translateBox->translateLine()->setFocus();
+      //    ui.translateBox->setText( t, true );
+      //    ui.translateBox->translateLine()->setCursorPosition( t.size() );
```

Comment:
What kind of change is this? Broken formatting and commented out code? This is clearly unready for review.